ocaml/typing/ctype.ml

```diff
@@ -5719,9 +5719,9 @@ let rec build_subtype env (visited : transient_expr list)
           if deep_occur_list ty tl1 then raise Not_found;
           set_type_desc ty
             (Tvar { name = None;
-                    jkind = Jkind.Primitive.value
+                    jkind = Jkind.Type.Primitive.value
```

Comment:
I'm surprised this doesn't need a conversion

Reply:
Same as above, oversight due to `Jkind.Type.t = Jkind.t`.